procedures/README.md

```diff
@@ -0,0 +1,41 @@
+# Procedures Utility
+
+This repository contains the utility code for adding value sets to the snowstorm server and to Bahmni procedures. More details can be found in [this](https://bahmni.atlassian.net/wiki/spaces/BAH/pages/3132686337/SNOMED+FHIR+Terminology+Server+Integration+with+Bahmni) Wiki link
+
+# Run
+
+1. This utility requires Node, npm.
+
+### One time installation:
+
+These steps need to performed ONLY the FIRST TIME you set up this code.
+
+1. Install node/npm (node version: 10.11.0). Preferably use nvm, so that you have control over which project uses which version of node. See:
+
+   * [how to install Node using nvm](https://github.com/nvm-sh/nvm).
+   * [how to install NodeJS on mac](https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681)
+
+### Environment Variables
+
+| Variable          | Description                     | Example                                                                       |
+| ----------------- | ------------------------------- | ----------------------------------------------------------------------------- |
+| SAVE_VALUESET_URL | Snowstorm server value set url | https://snowstorm.snomed.mybahmni.in/fhir/ValueSet                            |
```

Comment:
better name than SAVE_VALUESET_URL? why not just VALUESET_URL?

Reply:
This has been updated to VALUESET_URL